v2.1/reference/sql/statements/flush-privileges.md

```diff
@@ -35,6 +35,5 @@ Query OK, 0 rows affected (0.01 sec)
 
 ## 另请参阅
 
-* [GRANT](/reference/sql/statements/grant.md)
```

Comment:
Please check whether the correct version is `[GRANT](/reference/sql/statements/grant-privileges.md)`.